### PR TITLE
Added information about AWP Billing

### PR DESCRIPTION
**Pull request by Monstrofil**
There are some major changes in this patch:

- added information about Premium subscriptions
- images were reviewed, images that no longer used were removed from repository
- admin interface screenshots were updated
- {WIP} whmcs plugin docs linked

### CODE REVIEW

File: docs/cloudlinux-os-plus/README.md [hunk 1]
```diff
@@ -1467,17 +1481,12 @@ There are the following requirements to use AccelerateWP:
...
-By toggling the `Enable AccelerateWP for all users` an administrator provides end-users with AccelerateWP feature.
-Once the feature suite is enabled by administrator, end-users will see an AccelerateWP tab in their cPanel interface and be able to activate the optimization feature.
+In the _CloudLinux Manager → AccelerateWP_ tab an administrator provides the information about active features and users.
 
 #### Suites usage statistics
-Once at least one feature suite is enabled, the AccelerateWP usage statistics is shown.
+When AccelerateWP is enabled, the AccelerateWP usage statistics is shown.
 
-![](/images/AWPStats.png)
+![](./images/awp/AWPStats.png)
```
Comment (MariaMozgunova) [inline docs/cloudlinux-os-plus/README.md:1507]: The file `./images/awp/AWPStats.png` has the same contents as the `./images/awp/AWPStatsPremium.png`. Maybe on the first image, there should be no AWP Premium statistics?

File: docs/cloudlinux-os-plus/README.md [hunk 2]
```diff
@@ -1528,16 +1540,78 @@ To disable both suites:
 cloudlinux-awp-admin set-suite --suite=accelerate_wp,accelerate_wp_premium --disallowed --users=<username>
 ```
 
+### WHMCS billing
+
+CloudLinux developed it's own WHMCS plugin 
+which provides you AccelerateWP billing integration our of the box.
```
Comment (MariaMozgunova) [inline docs/cloudlinux-os-plus/README.md]: _out_ of the box

File: docs/cloudlinux-os-plus/README.md [hunk 3]
```diff
@@ -1528,16 +1540,78 @@ To disable both suites:
...
+```
+cloudlinux-awp-admin set-suite --suite=accelerate_wp_premium --allowed --users=<username>
+```
+
+This command makes two things:
+- allow user to install plugin, bypassing the upgrade window;
+- reports premium as being used to CLN, starting the billing cycle for hoster.
+
+
+When user terminates or downgrades plan, the following command must be exetured:
```
Comment (MariaMozgunova) [inline docs/cloudlinux-os-plus/README.md]: executed?

Comment (MariaMozgunova) [reply docs/cloudlinux-os-plus/README.md]: In the way it is currently written, it is not explicit who should execute the command below. Should it be the hoster, or it happens automatically?

File: docs/cloudlinux-os-plus/README.md [hunk 4]
```diff
@@ -1528,16 +1540,78 @@ To disable both suites:
...
+cloudlinux-awp-admin set-options --upgrade-url https://plan.upgrade/splash
+```
+
+![](./images/awp/AWPUpgradeLink.png)
+
+
+In order to remove the link, just set upgrade url to the empty one.
+
+```
+cloudlinux-awp-admin set-options --upgrade-url https://plan.upgrade/splash
```
Comment (MariaMozgunova) [inline docs/cloudlinux-os-plus/README.md]: Is this a typo here? According to the description, option `--upgrade-url` should be empty.

**Review by MariaMozgunova — CHANGES_REQUESTED** [review]

File: docs/cloudlinux-os-plus/README.md (hunk 1 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md:1507]: Done.,

File: docs/cloudlinux-os-plus/README.md (hunk 2 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md]: Done

File: docs/cloudlinux-os-plus/README.md (hunk 3 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md]: Done

File: docs/cloudlinux-os-plus/README.md (hunk 4 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md]: Done.

File: docs/cloudlinux-os-plus/README.md [hunk 5]
```diff
@@ -1556,16 +1573,78 @@ To disable both suites:
...
+```
+cloudlinux-awp-admin set-suite --suite=accelerate_wp_premium --default --users=<username>
+```
+
+### Setup upgrade URL for AccelerateWP Premium
+
+The upgrade window can be customized with link to the plan upgrade page, which can be set using CLI command:
+
+```
+cloudlinux-awp-admin set-options --upgrade-url https://plan.upgrade/splash
```
Comment (MariaMozgunova) [inline docs/cloudlinux-os-plus/README.md:1628]: I guess https://plan.upgrade/splash should be a string as well

Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md:1628]: It's fine, it works in bash both ways, 'string' and string.

File: docs/cloudlinux-os-plus/README.md [hunk 6]
```diff
@@ -1429,35 +1429,49 @@ Use this logging with caution because when it is enabled, the size of the daemon
...
 
-The Object Caching mechanism stores database query results in additional storage for quick access. This mechanism is really helpful in case if website needs to process multiple pages per second as requests come in and may be helpful in case when full page caching cannot be used, e.g. on personalized pages.
+#### AccelerateWP suite
+This is a basic suite which includes [AccelerateWP base feature](https://user-docs.cloudlinux.com/wpos-plugin/#acceleratewp-feature-wordpress-optimization-plugin): 
+a WordPress optimization plugin that provides full page caching, GZIP compression and some other useful optimizations.
+
+#### AccelerateWP Premium suite
 
 :::warning Attention
 This feature is free of charge only during the beta testing period. Afterwards it will be charged.
```
Comment (dar-ushka) [inline docs/cloudlinux-os-plus/README.md]: Should we add a concrete date here? April 2023

File: docs/cloudlinux-os-plus/README.md [hunk 7]
```diff
@@ -1556,16 +1573,78 @@ To disable both suites:
...
+
+![](./images/awp/AWPDisable.png)
 
 This operation will:
 * disable AccelerateWP tab in users' cPanel interface
 * disable all AccelerateWP optimization suites
 * deactivate all optimization features for all users
 
+This operation will NOT:
+* cancel the subscription made by user in WHMCS or other billing
```
Comment (dar-ushka) [inline docs/cloudlinux-os-plus/README.md]: Does it mean, that the hoster would still be billed for some users even when he stopped using AccelerateWP and has disallowed all users?
If not and this mean something like "user's subscription stays alive but deactivated for a period of time", may be we should formulate it a bit clearer?

File: docs/cloudlinux-os-plus/README.md (hunk 6 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md]: I'm not a fan of concrete dates, so I added the same message that we have in lvemanager:

> Beta testing period ends no earlier than March 2023.

File: docs/cloudlinux-os-plus/README.md (hunk 7 above)
Comment (Monstrofil) [reply docs/cloudlinux-os-plus/README.md]: It means that "user's subscription stays alive and user will still be billed in billing system unless hoster manually terminates them"

**Review by MariaMozgunova — APPROVED** [review]

**Pull request merged**